### PR TITLE
ci(bigtable): clean up the unwanted files and dirs from the conforman…

### PR DESCRIPTION
**Pull request by liujiongxin**
…ce testing

- - -
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/11988)

### CODE REVIEW

**Comment by liujiongxin** [comment]
My bad, without the flag `-modcache` , the `go clean` does nothing.

File: google/cloud/bigtable/ci/run_conformance_tests_proxy_bazel.sh [hunk 1]
```diff
@@ -35,14 +35,16 @@ bazel_args=("$@")
...
-nohup "${BAZEL_BIN}" run "${bazel_args[@]}" :cbt_test_proxy_main -- 9999 &
+nohup "${BAZEL_BIN}" run "${bazel_args[@]}" :cbt_test_proxy_main -- 9999 >/dev/null 2>&1 &
 proxy_pid=$!
 popd >/dev/null
 
 # Run the test
 pushd /var/tmp/downloads/cloud-bigtable-clients-test/tests >/dev/null
 go test -v -skip Generic_CloseClient -proxy_addr=:9999
 exit_status=$?
+# Remove the entire module cache, including unpacked source code of versioned dependencies.
```
Comment (devbww) [inline google/cloud/bigtable/ci/run_conformance_tests_proxy_bazel.sh:46]: Nit: Probably best to wrap the comment before the 80th column.

**Review by devbww — COMMENTED** [review]
Thanks for following up.

File: google/cloud/bigtable/ci/run_conformance_tests_proxy_bazel.sh [hunk 2]
```diff
@@ -35,14 +35,16 @@ bazel_args=("$@")
 # Building the binary can easily take more than the timeout limit.
 pushd "$(dirname "$0")/../test_proxy" >/dev/null
 "${BAZEL_BIN}" build "${bazel_args[@]}" :cbt_test_proxy_main
-nohup "${BAZEL_BIN}" run "${bazel_args[@]}" :cbt_test_proxy_main -- 9999 &
+nohup "${BAZEL_BIN}" run "${bazel_args[@]}" :cbt_test_proxy_main -- 9999 >/dev/null 2>&1 &
```
[the remark this answers is not in the document]
Comment (liujiongxin) [reply google/cloud/bigtable/ci/run_conformance_tests_proxy_bazel.sh:38]: I didn't give it much thought in the first place. Let me remove nohup then.

File: google/cloud/bigtable/ci/run_conformance_tests_proxy_bazel.sh (hunk 1 above)
Comment (liujiongxin) [reply google/cloud/bigtable/ci/run_conformance_tests_proxy_bazel.sh]: Done.

**Review by devbww — APPROVED** [review]

**Pull request merged**